src/workflow/ArcanistLandWorkflow.php

```diff
@@ -682,7 +724,8 @@ private function printPendingCommits() {
     echo pht("The following commit(s) will be landed:\n\n%s", $out), "\n";
   }
 
-  private function findRevision() {
+  public function getRevision() {
```

Comment:
Do not rename existing methods. Avoid change existing code.

I even found `findRevision` medhot called at 344 line of this file.